c/new-io.c

```diff
@@ -383,9 +383,6 @@ ptr S_close_fd(ptr file, IBOOL gzflag) {
  /* refuse to close stdin, stdout, and stderr fds */
   if (!gzflag && fd <= 2) return Strue;
 
- /* file is not locked; do not reference after deactivating thread! */
```

Comment:
Please leave the comment here, because it still applies.

Reply:
Comment restored